contrib/vcloud/util.py

```diff
@@ -0,0 +1,90 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import collections
+import os
+import sys
+
+import benchexec.util
```

Comment:
This highlights that we have now another module named `util`, which can be confusing (unfortunately, there are already two such modules in BenchExec itself). So if this module is actually needed, it should be renamed.

Reply:
renamed.